src/css/tabs/auxiliary.css

```diff
@@ -40,6 +40,15 @@
     background: #828885;
 }
 
+.tab-auxiliary .mode.disabled .info {
+    background: #ff0000;
+    color: white;
```

Comment:
Now that we have defined variables, maybe is better to use them when possible:
```css   
   background: var(--error)
   color: var(--quietText)
```

Reply:
Updated.